user-guide/Basic_Functionality/Alarms/Working_with_alarms/Advanced_analytics_features/Behavioral_anomaly_detection/Providing_feedback_on_behavioral_anomalies.md

```diff
@@ -32,29 +55,31 @@ Feedback can be provided via the ![Feedback](~/user-guide/images/Feedback_Column
 
 To provide feedback:
 
-1. In the Alarm Console, open the *Anomalies* or *Suggestion events* tab. See [Navigating the light bulb menu](xref:Light_Bulb_Feature#navigating-the-light-bulb-menu) and [Adding a tab](xref:ChangingTheAlarmConsoleLayout#adding-a-tab).
+1. In the Alarm Console, open a tab that supports user feedback (*Anomalies*, *Suggestion events*, or *Predicted alarms* tab).
```

Comment:
This suggests that you can only give feedback in these tabs. However, you can give feedback in any tab that contains alarms or suggestion events from the analytics features that support it (even history tabs for instance). Only these three tabs above show the columns by default. For other tabs, you will have to add the columns manually (right-click the header and select 'Add/Remove column' > 'Actions' > 'Feedback'